cmd/kops/create_cluster.go

```diff
@@ -1039,3 +1050,22 @@ func parseCloudLabels(s string) (map[string]string, error) {
 	}
 	return m, nil
 }
+
+// setOverrides sets override values in the spec
+func setOverrides(overrides []string, cluster *api.Cluster, instanceGroups []*api.InstanceGroup) error {
+	for _, override := range overrides {
+		kv := strings.SplitN(override, "=", 2)
+		if len(kv) != 2 {
+			return fmt.Errorf("unhandled override: %q", override)
```

Comment:
> unhandled override

Is this actually override is in the wrong format? How about

> --override must be in key=value format.  For example cluster.spec.nodePortAccess=0.0.0.0/0

Reply:
TBH I'm not yet sure what the overrides should look like.  They could be JSON, for example.

I figured I would guard it behind a feature flag, we can start using it to get test coverage, and we can see what "feels" right.